local-cluster/tests/local_cluster.rs

```diff
@@ -3515,22 +3669,47 @@ fn generate_account_paths(num_account_paths: usize) -> (Vec<TempDir>, Vec<PathBu
 }
 
 struct SnapshotValidatorConfig {
-    _snapshot_dir: TempDir,
+    bank_snapshots_dir: TempDir,
     snapshot_archives_dir: TempDir,
     account_storage_dirs: Vec<TempDir>,
     validator_config: ValidatorConfig,
 }
 
 fn setup_snapshot_validator_config(
-    snapshot_interval_slots: u64,
+    snapshot_interval_slots: Slot,
     num_account_paths: usize,
 ) -> SnapshotValidatorConfig {
+    setup_snapshot_validator_config_with(
+        snapshot_interval_slots,
+        Slot::MAX,
+        snapshot_interval_slots,
+        num_account_paths,
+    )
+}
+
+fn setup_snapshot_validator_config_with(
```

Comment:
nit: Should we just move this into a constructor for `SnapshotValidatorConfig`?

Reply:
Sure, can do!

Reply:
Done!